src/apiClient.tsx

```diff
@@ -12,361 +12,379 @@ import {
   UseMutateProps,
 } from "restful-react";
 
-export const SPEC_VERSION = "0.1.0";
-export interface AddressReadProcedureResponse {
+export const SPEC_VERSION = "1.0.0";
+export interface AddressResponse {
   id: number;
-  line1: string;
-  line2: string;
-  city: string;
+  line1?: string;
+  line2?: string;
+  city?: string;
   province: string;
   postcode: string;
-  latitude: number;
-  longitude: number;
-  geohash: string;
   created_at: string;
 }
 
-export interface EntriesResponse {
-  id: number;
-  numberAvaliable: number;
-  numberTotal: number;
-  date: string;
-  location: LocationsResponse;
-  vaccine: number;
-  inputType: number;
-  tags_optional: string;
-  tags_required: string;
-  created_at: string;
+export interface GeneralResponse {
+  success: boolean;
+  data?: string;
 }
 
 export interface HTTPValidationError {
   detail?: ValidationError[];
 }
 
-export interface LocationsResponse {
+/**
+ * An enumeration.
+ */
+export type InputTypeEnum = number;
+
+export interface LocationExpandedResponse {
   id: number;
   name: string;
-  organization: string;
-  phone: string;
-  notes: string;
-  address: AddressReadProcedureResponse;
+  phone?: string;
+  notes?: string;
   active: number;
-  postcode: string;
+  postcode?: string;
+  url?: string;
+  tags?: string;
   created_at: string;
+  organization?: number;
+  address?: AddressResponse;
 }
 
-export interface OrganizationsCreateProcedureRequest {
-  full_name: string;
+export interface OrganizationCreateRequest {
+  full_name?: string;
   short_name: string;
-  description: string;
+  description?: string;
+  url?: string;
 }
 
-export interface OrganizationsReadProcedureResponse {
+export interface OrganizationResponse {
   id: number;
-  full_name: string;
+  full_name?: string;
   short_name: string;
+  description?: string;
+  url?: string;
+  created_at: string;
+}
+
+export interface RequirementResponse {
+  id: number;
+  name: string;
   description: string;
   created_at: string;
 }
 
+export interface VaccineAvailabilityExpandedResponse {
+  id: string;
+  numberAvaliable: number;
```

Comment:
numberAvailable is spelled wrong here, which leads to it being undefined. Changing this to correct spelling resolved it.